### PR TITLE
Enhancement :Add prettier to project

### PR DESCRIPTION
**Pull request by harshilparmar**
As discussed in #345,I have added prettier as a dev-dependency and made a script to format hooks.
Let me know if any other changes are needed 👍

#### Suggestion: It would be better to add a pre-commit hook that prettifies code before every commit.

### CODE REVIEW

**Comment by maciekgrzybek** [comment]
I would add a pre-push hook. Also would be a good idea to add `prettierrc` file to have the config there

**Comment by imbhargav5** [comment]
Superb! Can someone confirm that the package.json has nothing broken scripts wise?

**Comment by imbhargav5** [comment]
Also, thanks for the contribution @harshilparmar !! This helps a ton!

File: package.json
Comment (imbhargav5) [inline package.json:127]: Can you add this to dependencies?  @harshilparmar 

Devdependencies isn't of a great use in yarn workspaces and actually causes confusion when working in a CI env like GitHub actions. 

It's not a big deal but let's keep it alongside others in dependencies. It's in the root of the workspace anyway so it's no big deal and does not contribute to package size.

Comment (harshilparmar) [reply package.json:127]: Sure I will do it 🤝 But why its failing !! Due to this Devdependencies ??

Comment (imbhargav5) [reply package.json:127]: It's a faulty Github Action. Don't worry about it. The pull request build succeeded.

**Comment by maciekgrzybek** [comment]
Can you also add `.prettierrc` file? So we can have the config there. Also I think it would make sense to create a separate ticket where we format the whole project with the prettier rules. What do you think @imbhargav5 ?

**Comment by imbhargav5** [comment]
Sounds good to me.

**Comment by harshilparmar** [comment]
> 
> 
> Can you also add `.prettierrc` file? So we can have the config there. Also I think it would make sense to create a separate ticket where we format the whole project with the prettier rules. What do you think @imbhargav5 ?

Sorry, I forgot about this I will do it in the next commit.

**Comment by imbhargav5** [comment]
Yep I think once we add the config we can merge this @harshilparmar .

**Comment by imbhargav5** [comment]
@all-contributors please add @harshilparmar for code.

**Comment by harshilparmar** [comment]
> 
> 
> Yep I think once we add the config we can merge this @harshilparmar .

**Comment by maciekgrzybek** [comment]
@harshilparmar done :)

**Review by maciekgrzybek — APPROVED** [review]
Good job mate :)

**Comment by maciekgrzybek** [comment]
Actually, we forgot about the hook that will run it pre-push right?

**Comment by imbhargav5** [comment]
Thanks @harshilparmar !

**Comment by imbhargav5** [comment]
Thanks @harshilparmar ! 

Also @maciekgrzybek , so we want to run prettier on pre-push? Actually let's not do that in this PR. I don't know how this ties into CI CD stuff. 

Let's keep this manual for a bit longer?

**Comment by maciekgrzybek** [comment]
Yeah fair enough, let's create a separate issue for that and investigate

**Comment by imbhargav5** [comment]
Going to merge this!

**Comment by imbhargav5** [comment]
@all-contributors please add @harshilparmar for code

**Comment by harshilparmar** [comment]
> 
> 
> Good job mate :)

My pleasure, looking forward to learn from you guys with more contribution to this awesome repo  🤗

**Pull request merged**
